src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -1065,6 +1065,7 @@ namespace winrt::TerminalApp::implementation
         _ApplyStartupTaskStateChange();
 
         Jumplist::UpdateJumplist(_settings);
+        ::Microsoft::Console::Render::FontCache::Invalidate();

```

Comment:
Notes lost in rebase:

> This is not going to do the right thing once @zadjii-msft lands "content process" terminal hosting. Do we need to do this? Unless our new font cache messes up _system font loading_ i'd rather not...

> Which function gets called in the content processes when we reload the settings.json?

I want to understand why we're clearing the cache here, and what this cache means for Terminal being able to access system fonts that are installed after it is launched...